src/Jaeger/Samplers/HttpSamplingManager.cs

```diff
@@ -31,15 +39,15 @@ internal SamplingStrategyResponse ParseJson(string json)
 
         public async Task<SamplingStrategyResponse> GetSamplingStrategyAsync(string serviceName)
         {
-            string url = "http://" + _hostPort + "/?service=" + Uri.EscapeDataString(serviceName);
```

Comment:
Interesting, this wasn't using the `/sampling` endpoint previously?  The results from `/` endpoint are using legacy Thrift-to-JSON conversion that used numbers for enums, instead of strings. Have you tested this change with an agent? I don't think crossdock tests cover the sampling endpoint.

Reply:
I checked the code again. The enum is not checked at all, therefore it's not making problems. C# is deserializing the following data structure as property. So it's looking for the field names instead of the enum. But I will adjust the test cases to make sure this reflects the real situation.